server/src/utils.ts

```diff
@@ -154,6 +159,28 @@ export let parseDiagnosticLocation = (location: string): Range => {
 	}
 };
 
+let findLocationSeparator = (fileAndLocation: string) => {
+	// Exclude the two first letters in windows paths to avoid the first colon in eg "c:\\.."		
+	if (process.platform === "win32") {
+		return fileAndLocation.indexOf(":", 2);
+	} else {
+		return fileAndLocation.indexOf(":")
+	}
+}
+
+let separateFileAndLocation = (fileAndLocation: string): [string, string] => {
+	let locationSeparator = findLocationSeparator(fileAndLocation)
+	let file = fileAndLocation.slice(0, locationSeparator)
+	let location = fileAndLocation.slice(locationSeparator + 1)
+	if (process.platform === "win32") {
+		return [`file:\\\\\\${file}`, location];
```

Comment:
So what's this for?

Reply:
Somewhat related: I had a note about `file:` at https://github.com/rescript-lang/rescript-vscode/blob/20841214830a3a81140f63a8973ab5d08cfcd79f/server/src/utils.ts#L20. Not sure if relevant

Reply:
Yeah I think so too, I'll try it and see if that gives us the same result

Reply:
Seems to be working on my Ubuntu and Windows 10. Do you see the current approach (Uri.file(..)) to be an issue?

Reply:
I should be what @chenglou was talking about, the utility vscode has for handling files.